internal/controller/compute/statefulserverset/serverset_controller.go

```diff
@@ -148,3 +166,32 @@ func extractSSetFromSSSet(sSSet *v1alpha1.StatefulServerSet) *v1alpha1.ServerSet
 func getSSetName(cr *v1alpha1.StatefulServerSet) string {
 	return fmt.Sprintf("%s-%s", cr.Name, cr.Spec.ForProvider.Template.Metadata.Name)
 }
+
+// Delete - deletes the server k8s client and waits until it is deleted
+func (k *kubeServerSetController) Delete(ctx context.Context, name, namespace string) error {
```

Comment:
Should this also be added to the `kubeSSetControlManager` interface?

Reply:
Not used externally, so not required. Might be better to be lowercased